From 61e2d7b01a9e5d8f3d6aa488ce81263f2223c89e Mon Sep 17 00:00:00 2001
From: Luca Stefani <luca.stefani.ge1@gmail.com>
Date: Tue, 5 Mar 2019 09:08:16 +0100
Subject: [PATCH 5/5] [DNM] Squash of lineage-16.0-android-9.0.0_r34

Squashed commit of the following:

commit 51b00a10cc3ab5cafb9d90909ef6dda8853738b5
Author: android-build-team Robot <android-build-team-robot@google.com>
Date:   Thu Jan 17 21:08:49 2019 +0000

    Version bump to PQ2A.190305.002

    Change-Id: I746b52e22fa429f61880066344481cd872784787

commit 4fde3a771e966d79d3a86383f394953e34fb2754
Merge: 6d7ba6e93 4ce3b412d
Author: android-build-team Robot <android-build-team-robot@google.com>
Date:   Wed Jan 16 23:03:48 2019 +0000

    Make change and version bump to PQ2A.190305.001

    Change-Id: I1ca7c33f8720d5f22555de2c557b77a1e887d508

commit 4ce3b412d822986feb0f4d6b35f93e30933386c0
Author: Adam Seaton <aseaton@google.com>
Date:   Tue Jan 8 14:29:13 2019 -0800

    Update Platform Security String to 2019-03-05 for pi-dev Bug:122541777
    (cherry picked from commit ac6eac206d0e4b0604a362d1bcbfae3e1b03bfc2)

    Change-Id: I2a39815b3589870e8d0cff64d6447a9fc1946d19

commit 6d7ba6e935e4d61fc2ee315f036d2a93d01c2da6
Author: android-build-team Robot <android-build-team-robot@google.com>
Date:   Wed Jan 16 18:58:26 2019 +0000

    Version bump to PQ2A.190301.002

    Change-Id: I3762c536366dc98cd930f823471fcb2ad29f1b58

commit 12567b09b372c83a412e62cbdf17b4cc85466912
Merge: b36e11d62 f627655b7
Author: android-build-team Robot <android-build-team-robot@google.com>
Date:   Tue Jan 15 20:59:09 2019 +0000

    Make change and version bump to PQ2A.190301.001

    Change-Id: Iddbd195f7151209b98bcf494c999311d8a5355d7

commit f627655b775fa7ad09c7b7316bc3472f20198c08
Author: Adam Seaton <aseaton@google.com>
Date:   Tue Jan 8 13:59:01 2019 -0800

    Update Platform Security String to 2019-03-01 for pi-dev
    Bug:122541777
    (cherry picked from commit a1165c3723c9d58afc83202ff63c6f691737effb)

    Change-Id: I631f45a270ac830c82d3360b73061a2bb73d1bd5

commit b36e11d62f1f68732aa9e0d1f0397ad10aba2a6b
Author: android-build-team Robot <android-build-team-robot@google.com>
Date:   Thu Dec 13 18:28:31 2018 +0000

    Version bump to PQ2A.190205.003

    Change-Id: I98f389c32af1e680a6652f38a49841aad2c70319

commit 9bba258684bd43c15df925dce38f3022db039b6c
Author: android-build-team Robot <android-build-team-robot@google.com>
Date:   Wed Dec 5 21:17:21 2018 +0000

    Version bump to PQ2A.190205.002

    Change-Id: I777d30e644ed2d17d45ccd51f3ac8f779481f7a8

Change-Id: Iccda3c787ce35b58b9261733ca11f4997621534a
---
 core/build_id.mk         | 2 +-
 core/version_defaults.mk | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/core/build_id.mk b/core/build_id.mk
index a1a64e32e..a6d63c858 100644
--- a/core/build_id.mk
+++ b/core/build_id.mk
@@ -18,4 +18,4 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=PQ2A.190205.001
+export BUILD_ID=PQ2A.190305.002
diff --git a/core/version_defaults.mk b/core/version_defaults.mk
index 57e5492f9..d386a527f 100644
--- a/core/version_defaults.mk
+++ b/core/version_defaults.mk
@@ -231,7 +231,7 @@ ifndef PLATFORM_SECURITY_PATCH
     #  It must be of the form "YYYY-MM-DD" on production devices.
     #  It must match one of the Android Security Patch Level strings of the Public Security Bulletins.
     #  If there is no $PLATFORM_SECURITY_PATCH set, keep it empty.
-      PLATFORM_SECURITY_PATCH := 2019-02-05
+      PLATFORM_SECURITY_PATCH := 2019-03-05
 endif
 
 ifndef PLATFORM_SECURITY_PATCH_TIMESTAMP
-- 
2.17.1

